instrumentation/cassandra/cassandra-4.0/src/main/java/io/opentelemetry/auto/instrumentation/cassandra/v4_0/TracingCqlSession.java

```diff
@@ -232,36 +224,36 @@ public ResultSet execute(@NonNull String query) {
   @NonNull
   public CompletionStage<AsyncResultSet> executeAsync(@NonNull Statement<?> statement) {
     final String query = getQuery(statement);
-    final Span span = startSpan(query);
 
-    try (final Scope scope = currentContextWith(span)) {
+    final Span span = TRACER.startSpan(session, query, null);
```

Comment:
```suggestion
    final Span span = TRACER.startSpan(session, query, /* originType= */ null);
```

Though using null origin seems to be a pattern, maybe can have an overload

Reply:
Let it leave be for a moment and we will address it in #689